pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -628,6 +628,15 @@
      */
     ConsumerBuilder<T> autoUpdatePartitionsInterval(int interval, TimeUnit unit);
 
+    /**
+     * Sets the interval of checking whether seek is complete interval.
+     *
+     * @param interval
+     *            the interval of check ticket.
+     * @return the consumer builder instance
+     */
+    ConsumerBuilder<T> seekCompleteCheckTicketMs(int interval);
```

Comment:
the name "seekCompleteCheckTicketMs" is very confusing. A better name is "seekCompletionPollIntervalMs" etc. . I wonder if the timeunit solution should be used to be consistent with other methods that configure time intervals.